test/pbf.test.js

```diff
@@ -74,12 +76,12 @@ test('readVarint64', function(t) {
 test('readVarint & writeVarint handle really big numbers', function(t) {
     var buf = new Pbf();
     var bigNum1 = Math.pow(2, 60);
-    var bigNum2 = Math.pow(2, 69);
```

Comment:
This is not a valid number to serialize in `varint`. The 64th bit on a signed integer is the sign bit. If we try to use the full 10th bytes, there is no way to distinguish the sign bit from the rest of the number.